### DISCUSSION TITLE
How do you add changes in AIR\lib\android\lib\resources\app_entry\res\values\styles.xml ?

### DISCUSSION THREAD

**Discussion by Mintonist** [airsdk/Adobe-Runtime-Support#3589] [Q&A]
Hi!

After each AIR update I add a string
`<item name="android:forceDarkAllowed">false</item>`
to AIR\lib\android\lib\resources\app_entry\res\values\styles.xml

Sometimes I forget about this( 
It is needed for my game.

Is there another way to do this? In description.xml or package as res folder or anything else? 
Thanks!

**Comment by ajwfrost** [comment]
At present, I don't think we have a way to override this..

We could potentially add an option to add items to the standard theme? which would basically mean you could do this within your app descriptor and it would then work across any future SDK update.

Alternatively it might we could allow a custom theme to be specified within the main activity, via the existing `manifestAdditions` section (where currently if you try setting the activity theme, it will say that `attribute android:theme is not allowed to be overridden`). Not sure whether there are side-effects to that though..

thanks

**Comment by Mintonist** [comment]
It will be great to any offer of yours)
May be somebody else can share his ideas?

**Comment by Mintonist** [comment]
@ajwfrost can you give example what I need to do in my case (I added `<item name="android:forceDarkAllowed">false</item>` to styles.xml) in lastest AIR 51.2.1.4?

**Reply by ajwfrost** [reply]
With this change, you can create your own styles.xml as part of your project (and add it via `resdir` or [similar](https://airsdk.dev/docs/tutorials/platform/android/custom-resources)) and then have contents such as:
```
<resources>
    <style name="Theme.MyBespokeTheme" parent="android:style/Theme.NoTitleBar">
        <item name="android:windowContentOverlay">@null</item>
        <item name="android:forceDarkAllowed">false</item>
    </style>
</resources>
```
and then set this up in your app descriptor within the main theme:
```
	<android>
		<manifestAdditions><![CDATA[
		  <manifest>
			<application>
			  <activity android:theme="@style/Theme.MyBespokeTheme">
				<intent-filter>
					<action android:name="android.intent.action.MAIN"></action>
					<category android:name="android.intent.category.LAUNCHER"></category>
				</intent-filter>
			  </activity>
			</application>
			<uses-permission android:name="android.permission.INTERNET"/>
		  </manifest>			
		]]></manifestAdditions>
	</android>
```

**Reply by marchbold** [reply]
@ajwfrost Brilliant!!!!

**Reply by Mintonist** [reply]
@marchbold I think we need to add this to https://airsdk.dev/docs

**Reply by marchbold** [reply]
Good idea, I have a few other things in my todo list to add so will add this one in there.
